internal/services/watch_btc_events.go

```diff
@@ -405,6 +418,7 @@ func (s *Service) handleSpendingUnbondingTransaction(
 			types.SubStateEarlyUnbondingSlashing,
 		)
 	}
+	// todo check
```

Comment:
@jeremy-babylonlabs I don't remember do we need to check that expansion here as well?